zenodo/modules/frontpage/templates/zenodo_frontpage/index.html

```diff
@@ -44,7 +44,7 @@
         {%- set publication_date = r.publication_date|from_isodate -%}
         {%- set creation_date = r._created|from_isodatetime -%}
         {%- set embargo_date = r.embargo_date|from_isodate -%}
-        {%- set object_type = r.resource_type|objecttype %}
+        {%- set object_type = r.resource_type.to_dict()|objecttype %}
```

Comment:
Elasticsearch-dsl uses an AttrDict which does have a get method. I added this here for now but the proper fix should most likely go into elasticsearch-dsl
https://github.com/elastic/elasticsearch-dsl-py/blob/master/elasticsearch_dsl/utils.py#L76